### DISCUSSION TITLE
Welcome to micropython-ulab Discussions!

### DISCUSSION THREAD

**Discussion by v923z** [v923z/micropython-ulab#571] [Announcements]
### Here are some useful tips:

1. Anyone can post and reply to discussions, you just need a GitHub account.
2. Please search before posting new issues.
3. You can use [GitHub Flavoured Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/about-writing-and-formatting-on-github) to format your posts. In particular use ```py ... ``` around your code snippets to get syntax highlighting, see the section below.
4. When asking your question, please describe what you've already tried. Include code snippets where possible!

**Comment by GodfriedDamm** [comment]
Newbie here.

I would very much like to use Ulab on my Pico 2W. However I'm unable to compile using the given instructions (sorry). Can someone please help and add the version for Pico 2W to the list of precompiled UF2's?

Thanks for any help!
